### ISSUE TITLE
How does WARCreate handle HTTP/2 communication?

### ISSUE THREAD

**Issue by machawk1** [machawk1/warcreate#103]
The current behavior needs to be documented. Per the WARC/1.1 spec, there is no documented "right way" but identifying the current approaches would be useful and help to guide how it might be done.

**Comment by ibnesayeed** [comment]
For the reference https://github.com/iipc/warc-specifications/issues/15

**Comment by machawk1** [comment]
That discussion was part of the impetus for creating this issue. This issue is about determining what the current behavior is. The result of that can be cross-referenced with other discussions on the correct representation.

**Comment by N0taN3rd** [comment]
@machawk1 I would assume the "right" way would be to turn the HTTP/2 protocols into HTPP/1.1. 

Reasoning: 
-  currently no existing replay system handles replay of HTTP/2
-  tools like warcreate and Squidwarc see headers as HTTP/2 but see the bodies as if they were sent over HTTP/1.1 (not the individule parts of the stream)

**Comment by machawk1** [comment]
@N0taN3rd Maybe a conversion record is more suitable for the HTTP/1.1 derivative while still maintaining the original payload while replay systems (and the WARC spec) catch up.

**Comment by ibnesayeed** [comment]
I would agree with @machawk1, because as an archivist you never want to lose information that might be useful later just because current tools do not support something.
